src/styles/block/calendar.scss

```
@@ -0,0 +1,37 @@
+.calendar {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  flex-shrink: 0;
+  gap: 1px;
+  padding: $calendar-padding;
+  max-width: mxwd($day-size, $calendar-padding, $day-border);
+  &__day {
+    height: $day-size;
+    width: $day-size;
+    cursor: pointer;
+    @extend %flex-center;
+    background-color: $day-bg-color;
+    border: 1px solid $border-color;
+    transition: background-color 0.5s, transform 0.5s;
+    @for $DayIndex from 1 through 31 {

```

Add empty lines between such blocks of code.
```suggestion
    transition: background-color 0.5s, transform 0.5s;
    
    @for $DayIndex from 1 through 31 {

```